app/mailers/user_mailer.rb

```diff
@@ -7,57 +7,16 @@ def reset_password(user)
   end
 
   def email_confirmation(user)
+    #comment this line when previewing
```

Comment:
no, you don't need to comment this line just make the user unconfirmed in the mailer preview and remove this comment